generators/server/templates/src/main/resources/config/_application-prod.yml

```diff
@@ -181,6 +181,9 @@ jhipster:
             host: localhost
             port: 2003
             prefix: <%= baseName %>
+        logs: # report metrics in the logs
+            enabled: false
+            reportFrequency: 60 # in seconds
```

Comment:
does doing this every minute have any performance impact? especially in prod when enabled?